packages/documentation/pages/usage/components/filters.vue

```diff
@@ -183,9 +196,6 @@ export default defineComponent({
 							Kotti.Filters.Operation.Currency.LESS_THAN_OR_EQUAL,
 							Kotti.Filters.Operation.Currency.IS_EMPTY,
 						],
-						prefix: settings.value.prefix,
```

Comment:
what why did you drop it from the return

you still needed for the relevant fields (float)

Reply:
This is the definition for the Currency column in the usage file for filters. KtFilters uses step, prefix and suffix only when column type is FLOAT.

Reply:
so it's not used on template level directly?

Reply:
No, there are computed properties for step, prefix and suffix in FilterRow.vue